Italian/Guardare/guarda.md

```diff
@@ -0,0 +1 @@
+mi guardo intorno e cerco di capire cosa accade. Vedo le pareti che si restringono, dei simpatici, carini e piccoli elfi che ballano e cantano e..comprendo che i funghi "speciali" che ho mangiato ieri in realta' erano allucinogeni!
```

Comment:
This line is longer than 120 characters. Please break it up into multiple lines, since tools like `git diff` are more difficult to use when lines are too long.